README.md

```diff
@@ -70,8 +151,16 @@ Marsha is compiled by an LLM into tested software that meets the requirements de
 ## Roadmap
 
 - Improve average accuracy for our test bed above 90%
-- Support for endpoints, visualizations, and data storage
-- Syntax highlighting
+- Support for visualizations and data storage (geek mode: handle side-effect logic better in general)
+- Syntax highlighting (vim, vscode, etc)
 - Support for different types of LLM
-- Being able to bootstrap the Marsha compiler with a Marsha program
-- More target languages other than Python
\ No newline at end of file
+- Bootstrap the Marsha compiler with a Marsha program
+- More target languages other than Python
+- A module system
```

Comment:
for marsha generated code?

Reply:
For code you write in Marsha -- if someone has written a very useful function in Marsha, it'd be nice to:

1. Get it.
2. Have it "pre-compiled" in Python (or whatever my target language is).

That last part on its own could be an NPM-like business where it houses Marsha code *and* pre-built "binaries" in various target languages to make such reuse valuable.